auth.go

```diff
@@ -7,8 +7,13 @@ import (
 	"go.uber.org/zap"
 )
 
-// ErrNotAllowed is returned if the user is not allowed access to a specific scope.
-const ErrNotAllowed = Error("not allowed")
+const (
+	// ErrNotAllowed is returned if the user is not allowed access to a specific scope.
+	ErrNotAllowed = Error("not allowed")
+
+	// PermissionKeyPrefix is a constant prefix appended to a userID
+	PermissionKeyPrefix = "joe.permissions."
```

Comment:
I think until we come up with a use case I would rather not yet export this constant.

Reply:
I've unexported the constant in e3bfb00456ea6e87386c40880fa3cd2baaf4fbc7